Source/IntegrationTests/TestFiles/LitTests/LitTest/comp/rust/continue.dfy

```diff
@@ -0,0 +1,38 @@
+// NONUNIFORM: Temporary development of the Rust compiler
```

Comment:
I'd replace these comments with something like "rust-specific tests". Otherwise I can just imagine finding this "Temporary" years later and being just a bit embarrassed. :)

Nothing wrong with having some tests written for specific backends - we still have `c++` tests after all.

Although we should also follow up with running these tests on all backends with `%testDafnyForEachCompiler` soon, as that will check that Rust behaves consistently. We can run all rust tests with and without `--raw-pointers` just as we run all C# tests with and without `--include-runtime`

Reply:
Just on top of my mind, it occurs to me that, whenever we run a Rust test, I believe it fetches all dependencies directly from the internet. The target folder weights 1.5Gb every time. This worries me for the CI capacity but I don't know if we can do something about it.

Reply:
I probably don't want to run tests with the two possible option values for `--raw-pointers`, because most tests don't have pointers, so that would be wasting CI resources.
But I'll replace temporary with something more appropriate